external_libraries/ghc/filesystem.hpp

```diff
@@ -102,6 +102,7 @@
 #ifdef GHC_EXPAND_IMPL
 
 #ifdef GHC_OS_WINDOWS
+#define NOMINMAX
```

Comment:
can you please revert the changes in this file? Should no longer be necessary

Reply:
Are you sure? Windows build was failing because of not having this line...

Reply:
you need to exclude the cpp that includes this from Cotire

in any case including this header in another header is nogo bcs it includes `windows.h`

Reply:
Hmm... I am not including this header in another header, but I am including this header in another cpp.... I'll try removing changes.

Reply:
done :)